eclipse-che/che-operator
# feat: propagate openshift console URL to dashboard

### What does this PR do?
feat: propagate openshift console URL to dashboard

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?
it's for https://github.com/eclipse/che/issues/20106

### How to test this PR?
Deploy che on OpenShift and verify that openshift console URL is propagated to dashboard deployment.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
